.github/workflows/test.yaml

```diff
@@ -22,6 +22,7 @@ env:
   TEST_BASE_URL: https://127.0.0.1/dashboard
   API: https://127.0.0.1
   TEST_PROJECT_ID: rancher-dashboard
+  TEST_DISABLE_DASHBOARD_LABEL: "${{ contains(github.event.pull_request.labels.*.name, 'ci/skip-e2e-dashboard')}}"
```

Comment:
`ci/skip-e2e-dashboard` might suggest it will skip the e2e test for rancher/dashboard. to avoid confusion can this be called something like `ci/skip-cypress-dashboard`?

Reply:
I would rather prefer name consistency, so either `skip-cypress` or `skip-e2e`.
2 names for the same thing really not.